lib/galaxy/tools/parameters/basic.py

```diff
@@ -307,7 +307,7 @@ def __init__( self, tool, input_source ):
             except:
                 raise ValueError( "An integer is required" )
         elif self.value is None and not self.optional:
-            raise ValueError( "The settings for the field named '%s' require a 'value' setting and optionally a default value which must be an integer" % self.name )
+            raise ValueError( "The settings for the field named '%s' require a 'value' attribute - a default value which must be an integer" % self.name )
```

Comment:
@jmchilton My reading is actually that the field is not optional, so you can probably drop " - a default value..." as well?